src/main/dumpxdr.cpp

```diff
@@ -64,37 +65,53 @@ dumpstream(XDRInputFileStream& in, bool compact)
 void
 dumpXdrStream(std::string const& filename, bool compact)
 {
-    std::regex rx(".*(ledger|bucket|transactions|results|meta|scp)-.+\\.xdr");
+    std::regex rx(
+        R"(.*\b(debug-tx-set|(?:(ledger|bucket|transactions|results|meta-debug|scp)-.+))\.xdr$)");
     std::smatch sm;
     if (std::regex_match(filename, sm, rx))
     {
         XDRInputFileStream in;
         in.open(filename);
 
-        if (sm[1] == "ledger")
+        if (sm[1] == "debug-tx-set")
         {
-            dumpstream<LedgerHeaderHistoryEntry>(in, compact);
+            dumpstream<StoredDebugTransactionSet>(in, compact);
         }
-        else if (sm[1] == "bucket")
+        else if (sm.size() == 3)
         {
-            dumpstream<BucketEntry>(in, compact);
-        }
-        else if (sm[1] == "transactions")
-        {
-            dumpstream<TransactionHistoryEntry>(in, compact);
-        }
-        else if (sm[1] == "results")
-        {
-            dumpstream<TransactionHistoryResultEntry>(in, compact);
-        }
-        else if (sm[1] == "meta")
-        {
-            dumpstream<LedgerCloseMeta>(in, compact);
+            auto& m2 = sm[2];
+            if (m2 == "ledger")
+            {
+                dumpstream<LedgerHeaderHistoryEntry>(in, compact);
+            }
+            else if (m2 == "bucket")
+            {
+                dumpstream<BucketEntry>(in, compact);
+            }
+            else if (m2 == "transactions")
+            {
+                dumpstream<TransactionHistoryEntry>(in, compact);
+            }
+            else if (m2 == "results")
+            {
+                dumpstream<TransactionHistoryResultEntry>(in, compact);
+            }
+            else if (m2 == "meta-debug")
```

Comment:
I think just "meta" is a bit more intuitive (and less verbose)

Reply:
this is the file name pattern, we could rename files, but this is out of scope for this PR